common/src/main/java/tachyon/util/io/FileUtils.java

```diff
@@ -30,6 +30,11 @@
 import tachyon.TachyonURI;
 import tachyon.thrift.InvalidPathException;
 
+/**
+ * Provides utility methods for working with files and directories.
+ * 
+ * Incoming methods please better take pathname string as parameters for standardization.
```

Comment:
Please reword this to something like: "By convention, methods take file path strings as parameters."